receiver/googlecloudspannerreceiver/internal/metadata/metricsdatapoint_test.go

```diff
@@ -111,6 +113,35 @@ func TestMetricsDataPoint_CopyTo(t *testing.T) {
 	}
 }
 
+func TestMetricsDataPoint_HideLockStatsRowrangestartkeyPII(t *testing.T) {
+	btSliceLabelValueMetadata, _ := NewLabelValueMetadata("row_range_start_key", "byteSliceLabelColumnName", StringValueType)
+	labelValue1 := byteSliceLabelValue{metadata: btSliceLabelValueMetadata, value: "table1.s(23,hello,23+)"}
+	labelValue2 := byteSliceLabelValue{metadata: btSliceLabelValueMetadata, value: "table2(23,hello)"}
```

Comment:
Can you add a few more cases to ensure it doesn't panic if it isn't properly formatted?  In particular, one without a `(`, and an empty string would be good to test.

Reply:
Done. Added cases without a "(", without a "," , and an empty string

Reply:
One more case :).  Can you add the string `(`?

Reply:
Done! Thanks for catching this case :)